pandas/tests/base/test_constructors.py

```diff
@@ -43,18 +43,18 @@ def constructor(request):
 
 class TestPandasDelegate:
     class Delegator:
-        _properties = ["foo"]
-        _methods = ["bar"]
+        _properties = ["prop"]
+        _methods = ["test_method"]
 
-        def _set_foo(self, value):
-            self.foo = value
+        def _set_prop(self, value):
+            self.prop = value
 
-        def _get_foo(self):
-            return self.foo
+        def _get_prop(self):
+            return self.prop
 
-        foo = property(_get_foo, _set_foo, doc="foo property")
+        prop = property(_get_prop, _set_prop, doc="foo property")
 
-        def bar(self, *args, **kwargs):
+        def test_method(self, *args, **kwargs):
             """a test bar method"""
```

Comment:
```suggestion
            """a test method"""
```

Or can just delete this